package.json

```diff
@@ -67,6 +67,7 @@
     "aws-sdk": "^2.642.0",
     "check-node-version": "^4.0.2",
     "concurrently": "^5.1.0",
+    "nan": "2.14.1",
```

Comment:
`npm test` fails to execute on some machines, due to `nan` not getting installed (optional dependency of a dependency?).

Reply:
Also mind you, I have `save-exact=true` set on my `.npmrc`, is this an issue?

Reply:
>due to nan not getting installed

Uhh, haven't noticed. :thinking: 

>is this an issue?

I don't mind, I'm just too lazy to do that myself. A lockfile is 99,9% of the solution to this problem anyway.